packages/core/echo/echo-schema/src/document.ts

```diff
@@ -137,7 +137,7 @@ export class DocumentBase extends EchoObject<ObjectModel> {
   }
 
   private _getModelProp(prop: string): any {
-    let type = this._item!.model.get(`${prop}$type`);
+    let type;
```

Comment:
For serialization (both toJSON and import/export) we need to be able to inspect (recursively) the object model (Document) properties.